solutions/observability/streams/management/data-quality.md

```diff
@@ -0,0 +1,29 @@
+---
+applies_to:
+  serverless: preview
+  stack: preview 9.1, ga 9.2
+---
+
+# Manage data quality [streams-data-retention]
+
+Use the **Data quality** tab to find failed and degraded documents in your stream. Use the following components to monitor the health of your data and identify and fix issues:
+
+- **Degraded documents:** Documents with the `ignored` property usually because of malformed fields or exceeding the limit of total fields when `ignore_above:false`. This component shows the total number of degraded documents, the percentage, and status (**Good**, **Degraded**, **Poor**).
```

Comment:
Not sure if these edits are correct. The first sentence seems a bit oddly constructed.

```suggestion
- **Degraded documents:** Documents with the `ignored` property usually have malformed fields or exceed the limit of total fields when `ignore_above:false`. This component shows the total number of degraded documents, the percentage, and status (`Good`, `Degraded`, `Poor`).
```